odoo/addons/base/models/res_users.py

```diff
@@ -1484,6 +1484,13 @@ def _should_alert_new_device(self):
         """
         return False
 
+    def _alert_untrusted_location(self):
+        """
+            To be overriden in other modules to perform actions
+            if an unknown location is detected.
+        """
+        return
```

Comment:
https://pep8.org/#a-foolish-consistency-is-the-hobgoblin-of-little-minds
I did not make that title, Guido did, and I share his views.
```suggestion
        """ Deal with new untrusted geoip location detected during login.
        
        To be overriden.
        """
        pass
```